demos/drawer/drawer.scss

```diff
@@ -59,36 +60,48 @@
   padding: 10px;
 }
 
-.demo-drawer--custom {
-  @include mdc-drawer-fill-color($material-color-teal-900);
-  @include mdc-drawer-ink-color(#fff);
-  @include mdc-list-item-graphic-ink-color(#fff);
+.demo-form-field {
+  margin: 16px 0;
+}
 
-  .demo-drawer-list-item {
-    @include mdc-list-item-graphic-ink-color(text-icon-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-secondary-on-dark);
-  }
+/* Place drawer and content side by side. */
+.demo-body {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+}
 
-  .demo-drawer-list-item.mdc-list-item--selected {
-    @include mdc-list-item-graphic-ink-color(text-primary-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-primary-on-dark);
-  }
+.demo-content {
+  height: 100%;
+  width: 100%;
+  overflow: auto;
 }
 
-.demo-drawer--accessible {
-  @include mdc-drawer-fill-color-accessible($material-color-indigo-500);
+.demo-main {
+  padding-left: 16px;
+  padding-right: 16px;
+}
 
-  .demo-drawer-list-item {
-    @include mdc-list-item-graphic-ink-color(text-icon-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-secondary-on-dark);
-  }
+#extra-wide-content {
+  width: 200vw;
+}
 
-  .demo-drawer-list-item.mdc-list-item--selected {
-    @include mdc-list-item-graphic-ink-color(text-primary-on-dark);
-    @include mdc-list-item-primary-text-ink-color(text-primary-on-dark);
-  }
+#extra-tall-content {
+  height: 200vh;
 }
 
-.demo-form-field {
-  margin: 16px 0;
+.demo-drawer--custom {
+  @include mdc-drawer-border-color(lightblue, $mdc-drawer-divider-opacity);
+  @include mdc-drawer-group-header-text-color(darkolivegreen, $mdc-drawer-subtitle-ink-opacity);
+  @include mdc-drawer-surface-fill-color(pink, $mdc-drawer-surface-fill-opacity);
+  @include mdc-drawer-icon-ink-color(darkolivegreen, $mdc-drawer-icon-ink-opacity);
+  @include mdc-drawer-title-ink-color(darkolivegreen, 1);
+  @include mdc-drawer-activated-overlay-color(darkcyan);
+  @include mdc-drawer-item-text-color(darkolivegreen, $mdc-drawer-inactive-ink-opacity);
+  @include mdc-drawer-item-activated-text-color(darkcyan, 1);
+
+  .mdc-list-item--selected {
```

Comment:
Putting aside that this should be activated rather than selected, why do we have a mixin for activated text color but not activated icon color? (See also question RE having a general item mixin and an icon mixin)